source/extensions/tracers/datadog/time_util.cc

```diff
@@ -0,0 +1,27 @@
+#include "source/extensions/tracers/datadog/time_util.h"
+
+#include <chrono>
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace Datadog {
+
+datadog::tracing::TimePoint estimateTime(SystemTime wall) {
+  return estimateTime(wall, datadog::tracing::default_clock);
+}
+
+datadog::tracing::TimePoint estimateTime(SystemTime wall, const datadog::tracing::Clock& clock) {
+  auto point = clock();
```

Comment:
can you give a precise type here? It's not obvious from context what type `point` is.

Reply:
Will do. It's `datadog::tracing::TimePoint`, the return value of `datadog::tracing::Clock`.